stdlib/public/Concurrency/Executor.swift

```diff
@@ -93,6 +93,31 @@ func _checkExpectedExecutor(_filenameStart: Builtin.RawPointer,
     _filenameStart, _filenameLength, _filenameIsASCII, _line, _executor)
 }
 
+@available(SwiftStdlib 5.1, *)
```

Comment:
Just an FYI, a function marked `@_alwaysEmitIntoClient` doesn't need availability.

Reply:
thanks!

Reply:
actually since `_taskIsCurrentExecutor` has that availability, I need to keep it in this case.